docs/tenancy/1.x/tutorial-non-tenant.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Non-Tenant Site Setup
+icon: fal fa-gears
+excerpt: >
+    How to setup goabal and admin sites in the same application
+tags:
+    - tenancy
+    - tutorial
+    - non-tenant
+---
+
+## Introduction
+When building a SaaS there tends to be a need for a "sub-site" such as an admin panel, or a public facing website describing your SaaS.
+
+This tutorial will go though the basics of setting these "sub-sites" within the same codebase without the need for creating a tenant do display these sites.
+
+## Requirements
+This tutorial will assume that you have followed the [basic setup tutorial](tutorial-basic-setup).
+
+## Routes
+The first thing we want to do is configure the [Routes Affect](affacts-routes).
+
+First start by creating a new file for all tenant specific routes.
```

Comment:
This is repeated from above. First we do, first we do that.

A more fluid way of saying this is something like

```suggestion
In order to set up the tenant specific routes, we will need to create a routes file for it.
```